docs/api/index.rst

```diff
@@ -0,0 +1,139 @@
+.. SPDX-License-Identifier: MIT
+
+.. currentmodule:: disnake
+
+.. _disnake_api_index:
+
+API Reference
+=============
+
+The following documents outline the API of disnake.
+
+.. note::
+
+    This module uses the Python logging module to log diagnostic and errors
+    in an output independent way.  If the logging module is not configured,
+    these logs will not be output anywhere.  See :ref:`logging_setup` for
+    more information on how to set up and use the logging module with
+    disnake.
+
+.. _version_related:
+
+Version Related Info
```

Comment:
ideally these should be on a seperate file, I don't want any objects documented in this file.

Reply:
Done in [`d15ddd7`](https://github.com/DisnakeDev/disnake/pull/722/commits/d15ddd70e20c611070a3445379763a80b07a8c13)